docker-compose.yml

```diff
@@ -21,6 +21,11 @@ services:
       - env/db.env
     ports:
       - "5555:5432"
+  redis:
+    image: redis
+    container_name: redis
+    ports:
+       - "127.0.0.1:16379:6379"
```

Comment:
Let's change it. 
To env (for backend) add `REDIS_HOST=redis://redis:6379`
Change here image redis:alpine, host name "redis", restart always

Don't open ip address as now in ports